nucypher/cli/characters/alice.py

```diff
@@ -35,6 +35,9 @@
 @click.option('--dev', '-d', help="Enable development mode", is_flag=True)
 @click.option('--force', help="Don't ask for confirmation", is_flag=True)
 @click.option('--dry-run', '-x', help="Execute normally without actually starting the node", is_flag=True)
+@click.option('--policy-encrypting-key', help="Encrypting Public Key for Policy as hexadecimal string", type=click.STRING)
+@click.option('--alice-verifying-key', help="Alice's verifying key as a hexadecimal string", type=click.STRING)
```

Comment:
Reassess if you need these two lines, knowing that you don't need the policy encrypting key nor alice's verifying to in order to decrypt by Alice.